### PR TITLE
Newsletter 13: Crate Before Attack

### PR DESCRIPTION
**Pull request by koalefant**
_Part of #232 ("Newsletter 13")_

### CODE REVIEW

File: content/posts/newsletter-013/index.md [hunk 1]
```diff
@@ -62,6 +62,36 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+
+[Crate Before Attack][cba-site] by [koalefant (@CrateAttack)][@CrateAttack]
+is a skill-based multiplayer game where frogs combat their friends
+while navigating the landscape with their sticky tongues.
+
+A [playable browser build][cba-play] can be tried online.
+
+Recent changes are:
+
+- Training mode improvements, including a new map [Dungeon][cba-youtube-dungeon] by [Kesha Astafyev][cba-spoon-tar].
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-013/index.md:79]:
```suggestion
- Training mode improvements, including a new map
  [Dungeon][cba-youtube-dungeon] by [Kesha Astafyev][cba-spoon-tar].
```

File: content/posts/newsletter-013/index.md [hunk 2]
```diff
@@ -62,6 +62,36 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+- [Better camera motion][cba-youtube-camera-motion]: multiple points of interest are tracked dynamically.
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-013/index.md:80]:
```suggestion
- [Better camera motion][cba-youtube-camera-motion]: multiple points of
  interest are tracked dynamically.
```

**Review by 17cupsofcoffee — APPROVED** [review]
Content looks good, couple of changes required to appease the linter.

**Review by ozkriff — APPROVED** [review]
:+1: (fixed fmt warns)

**Comment by ozkriff** [comment]
Merged, thanks for the PR!

**Pull request merged**